lib/extensions/queue-me.js

```diff
@@ -45,6 +46,26 @@ var queueMeExtension = function(keyword, playQueue) {
     };
 };
 
+function urlSanitiser(url){
+    url = url.replace(/[{()}]/g, '');
+    url = url.replace(/[\[\]']+/g, '');
+    return url;
+}
+
+function getService(url){
+    var youtubeURLs = ['youtu', 'youtube'];
+    var parsedURL = extensionHelper.parseURL(url);
+    if (youtubeURLs.indexOf(parsedURL.host) != -1) {
+        return 'youtube';
+    }
+}
+
+function getVideoId(url){
+    var regExp = /^.*((youtu.be\/)|(v\/)|(\/u\/\w\/)|(embed\/)|(watch\?))\??v?=?([^#\&\?]*).*/;
```

Comment:
since regExp never changes I'd move this one level higher, outside of getVideoId.